bees/common.py

```diff
@@ -9,25 +9,10 @@
 VERSION is the full ARC version, using `semantic versioning <https://semver.org/>`_.
 """
 
```

Comment:
This PR adds "common" and "logger", and the previous PR added "common" and "logger"